src/ApiPlatform/Resources/Module/BulkModules.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
```

Comment:
AFL not OSL

Reply:
Right but it's wrong in the whole module actually, I'll do a dedicated PR after to fix it all